### PR TITLE
Add kubelet, systemd filtration and refactor containers exporter

### PR DESCRIPTION
**Pull request by sumo-drosiek**
- unify exporting for systemd, collectd and containers logs
- add filtration for systemd like we already have in fluentd
- add kubelet pipeline

### CODE REVIEW

File: deploy/helm/sumologic/values.yaml [hunk 1]
```diff
@@ -2987,7 +2993,20 @@ otelcol:
...
-                - sumologic/systemd
+                - sumologic
+            logs/kubelet:
+              receivers:
+                - fluentforward
+              processors:
+                - memory_limiter
+                - filter/include_fluent_tag_host
+                - attributes/remove_fluent_tag
+                - filter/include_kubelet
```
Comment (pmalek-sumo) [inline deploy/helm/sumologic/values.yaml:3004]: Actually as I think of it now I believe we can swap those 2 to save processing of "non kubelet" entries, i.e. no need to remove the `fluent.tag` from a record that will be filtered out anyway.

The same can be done for systemd

Comment (sumo-drosiek) [reply deploy/helm/sumologic/values.yaml:3004]: so we should move it as far as possible, we have additional filtering in sourceprocessor for example

Comment (pmalek-sumo) [reply deploy/helm/sumologic/values.yaml:3004]: What do you mean "as far as possible"? Shouldn't we filter out as soon as possible in the pipeline?

Comment (sumo-drosiek) [reply deploy/helm/sumologic/values.yaml:3004]: I mean moving `- attributes/remove_fluent_tag` just before `batch`

**Review by pmalek-sumo — APPROVED** [review]

**Pull request merged**